docker/addons/bootstrap/docker-compose.yml

```diff
@@ -17,7 +17,7 @@ volumes:
 
 services:
   bootstrap-db:
-    image: postgres:10.2-alpine
+    image: postgres:10.8-alpine
```

Comment:
There is version 13 available, should with a try with the latest or there is a specific reason to use 10.x?

Reply:
Because version 13 doesn't support DBs initialised with version 10. We can probably do it for the next release

Reply:
Really? Damn. So now we will move from version 10 in the future? IMHO, it's docker-compose meant for local development, not really a breaking change (we can note it in release).

Reply:
We plan to upgrade but probably a better moment would be when we replace User email by User ID.

Reply:
Ok. Btw looks really easy to do [safe upgrade](https://medium.com/better-programming/how-to-upgrade-your-postgresql-version-using-docker-d1e81dbbbdf9) with few commands, for existing databases that want to keep data (even if its development setup).